### ISSUE TITLE
VERY VERY LOW AP VALUE AND OTHER EVALUATION INDEXES ON CUSTOM DATASETS

### ISSUE THREAD

**Issue by qtw1998** [google/automl#312]
![](https://site-pictures.oss-eu-west-1.aliyuncs.com/qtuwp.png)
Low params size caused  THE Hyperparameter sensitive, so the very little fluctuation in params settings would bring a low ap result.... Powerful computing sources are needed to find the proper settings.🤯

**Comment by mingxingtan** [comment]
Hmm, I am not sure about the details, but because the #params is much smaller, you generally need to train longer.

**Comment by qtw1998** [comment]
> Hmm, I am not sure about the details, but because the #params is much smaller, you generally need to train longer.

![image](https://user-images.githubusercontent.com/34357479/80306105-cdeebf80-87f3-11ea-9fdc-7d442c7e65c7.png)
I've trained it for quite a long time 3days plus....
and the loss never get down below 0.4 
![image](https://user-images.githubusercontent.com/34357479/80306132-f8407d00-87f3-11ea-9b48-fd6ab52164bf.png)

**Comment by qtw1998** [comment]
> Hmm, I am not sure about the details, but because the #params is much smaller, you generally need to train longer.

could you please share some tricks to attach better training courses like hyperparams settings  like anchor size changed after measuring the datasetting by kmeans.

**Comment by denisb411** [comment]
> > Hmm, I am not sure about the details, but because the #params is much smaller, you generally need to train longer.
> 
> could you please share some tricks to attach better training courses like hyperparams settings like anchor size changed after measuring the datasetting by kmeans.

Did you got better results by changing some hparam? Could you please share with us? I`m having the same `issue`.

**Comment by zishanahmed08** [comment]
@qtw1998 Any suggestion on hparam changes?

**Comment by qtw1998** [comment]
> @qtw1998 Any suggestion on hparam changes?

I've changed to pytorch version..
